style.css

```diff
@@ -8,10 +8,67 @@
   padding: 0 24px;
 }  */
 
-nav {
+.nav {
```

Comment:
[OPTIONAL] Unfortunately, the previous reviewer didn't notice, but let's add the bottom gap and border-radius like on the Figma design below.
<img width="363" alt="Screenshot 2023-01-30 at 10 59 35 PM" src="https://user-images.githubusercontent.com/25887253/215589073-0df14d88-ea48-4681-ad68-b2337069aa01.png">

Reply:
@huseyinbiyik thanks for spotting that for me, will add it as advised.